liquibase/src/main/java/io/micronaut/liquibase/AbstractLiquibaseMigration.java

```diff
@@ -67,6 +68,10 @@
     AbstractLiquibaseMigration(ApplicationContext applicationContext, ResourceAccessor resourceAccessor) {
         this.applicationContext = applicationContext;
         this.resourceAccessor = resourceAccessor;
+
+        // Bridge JUL to Slf4j
+        SLF4JBridgeHandler.removeHandlersForRootLogger();
+        SLF4JBridgeHandler.install();
```

Comment:
I added this here to enable the jul-slf4j bridge also the users have to add something manually on their `logback.xml` file. I added that to the docs.

Reply:
I'm not sure we should be doing this at this point, or even at all. It could be that there are already logs happening before this is executed. We should probably instruct users to do this as part of their Application class. That way they have the option of not doing it in the case that it causes a problem or if they don't care about the liquibase logs.

Reply:
I agree, alternatively maybe there should be some environment variable in core they could set to enable this behaviour but it doesn't belong in the liquibase support

Reply:
Removed and added to the docs